libs/sdk-backend-tiger/src/backend/uiSettings.ts

```diff
@@ -49,7 +49,8 @@ export const DefaultUiSettings: ISettings = {
     // drilling
     enableKPIDashboardDrillToDashboard: false,
     enableKPIDashboardDrillToInsight: false,
-    enableKPIDashboardDrillToURL: false,
+    // TODO: To be able to test on tiger, change it back.
+    enableKPIDashboardDrillToURL: true,
```

Comment:
add also its definition in settings.ts and check with Jakub possible conflict, he wanted to add this FF too